app/src/main/java/com/github/factotum_sdp/factotum/data/LoginDataSource.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.factotum_sdp.factotum.data
+
+import com.github.factotum_sdp.factotum.data.model.LoggedInUser
+import java.io.IOException
+
+/**
+ * Class that handles authentication w/ login credentials and retrieves user information.
+ */
+class LoginDataSource {
+
+    fun login(username: String, password: String): Result<LoggedInUser> {
+        try {
+            // TODO: handle loggedInUser authentication
```

Comment:
Is this something that had to be done before the merge ? Or a remainder for later ?

Reply:
It will be implemented in the next task